mpas_analysis/ocean/time_series_antarctic_melt.py

```diff
@@ -723,7 +723,8 @@ def run_task(self):
         # and cartopy doesn't play too well with tight_layout anyway
         plt.tight_layout()
 
-        add_inset(fig, fc, width=2.0, height=2.0)
+        #add_inset(fig, fc, width=2.0, height=2.0)
```

Comment:
```suggestion
```

Reply:
We don't generally leave commented-out code.  There's always the history of the file and the `blame` button to see what was there previously if folks want to go back.